mozilla/fx-private-relay
# MPP-2263 Phone Numbers tab redirect update

This PR fixes  MPP-2263
 



# New feature description

Clicking on the “Phone Numbers” tab redirected the user to the “Verify your phone number” screen instead of the purchase flow. This PR should make it so clicking on the tab redirects the user to the “Verify your phone number” screen of the flow instead.

# Screenshot (if applicable)

Not applicable.

# How to test

- Sign in into a Premium Firefox Relay account
- Go to the Relay dashboard
- Click on the “Phone Numbers” tab in the site header
- Observe the results


# Checklist
